### PR TITLE
tracing-flame: add module_path and file_and_line into Config

### PR DESCRIPTION
**Pull request by xiaopengli89**
We should allow config whether or not display module_path or file/line in output.

### CODE REVIEW

File: tracing-flame/src/lib.rs [hunk 1]
```diff
@@ -307,6 +315,18 @@ where
         self.config.threads_collapsed = enabled;
         self
     }
+
+    /// Configures whether or not display module path
```
Comment (yaahc) [inline tracing-flame/src/lib.rs:319]:
```suggestion
    /// Configures whether or not module paths should be included in the output.
```

File: tracing-flame/src/lib.rs [hunk 2]
```diff
@@ -307,6 +315,18 @@ where
...
+    pub fn with_module_path(mut self, enabled: bool) -> Self {
+        self.config.module_path = enabled;
+        self
+    }
+
+    /// Configures whether or not display file and line
```
Comment (yaahc) [inline tracing-flame/src/lib.rs:325]:
```suggestion
    /// Configures whether or not file and line should be included in the output
```

**Review by yaahc — COMMENTED** [review]
looks good, just want to correct some grammar in the doc comments

**Review by yaahc — APPROVED** [review]
tyvm  ^_^

File: tracing-flame/src/lib.rs [hunk 3]
```diff
@@ -307,6 +315,18 @@ where
...
         self
     }
+
+    /// Configures whether or not module paths should be included in the output.
+    pub fn with_module_path(mut self, enabled: bool) -> Self {
+        self.config.module_path = enabled;
+        self
+    }
+
+    /// Configures whether or not file and line should be included in the output
```
Comment (hawkw) [inline tracing-flame/src/lib.rs:325]: nit:
```suggestion
    /// Configures whether or not file and line should be included in the output.
```

**Review by hawkw — APPROVED** [review]
looks good to me, thank you!

**Pull request merged**